### PR TITLE
Document the need to create a CORS policy in S3

### LINKED ISSUE #209: Problem with display of officer picture in tagging app
Discovered while testing the upcoming 0.2 release. I uploaded one of our photographs without issue and proceeded to the classification volunteer task:

screen shot 2017-03-15 at 1 14 57 pm

I was able to classify the image as having a cop. Then I went to tag the photo (associate the photo with the correct officer) and the photograph was not displayed:

screen shot 2017-03-15 at 1 18 52 pm

However the right path exists in the source code:
```
<img class="img-responsive" id="image" src="https://s3-us-west-1.amazonaws.com/openoversight-staging/1f/59dd3046c8fd0b4d846385913202a50cc18eef9bf95a0be5b14c2d31c60d94.jpg" alt="Picture">
```

Not sure what is happening here on the tagging volunteer task, needs further investigation.

### PR DESCRIPTION
**Pull request by r4v5**
## Status

Ready for review

## Description of Changes

Updates docs to mention the CORS policy needed

Fixes #209.

Changes proposed in this pull request:

 - Docs

## Tests and linting
 
 [x] I have rebased my changes on current `develop`
 
 [n/a] pytests pass in the development environment on my local machine
 
 [n/a] `flake8` checks pass

### CODE REVIEW

File: DEPLOY.md [hunk 1]
```diff
@@ -35,6 +35,21 @@ You'll need to create an AWS account, if you don't already have one. Then, you'l
 }
 ```
 
+For the officer identification UI to work, you'll need to create a CORS policy for the S3 bucket used with OpenOversight. Since we're not doing anything fancier than making a web browser GET it, we can just use the default policy:
```
Comment (freddymartinez9) [inline DEPLOY.md]: For someone who has never set a CORS policy in S3, it is unclear where this seeing would be set. Needs a bit more documentation.

**Comment by r4v5** [comment]
@freddymartinez9 is the text clearer now?

**Comment by freddymartinez9** [comment]
@r4v5 looks good to me. Feel free to merge when the travis builds pass.

**Comment by r4v5** [comment]
Ah yes, lest my change to DEPLOY.md break our python code.

**Pull request merged**